libra_swarm/src/swarm.rs

```diff
@@ -133,7 +122,7 @@ impl LibraNode {
             if num_connected_peers != expected_peers {
                 debug!(
                     "Node '{}' Expected peers: {}, found peers: {}",
-                    self.peer_id, expected_peers, num_connected_peers
+                    self.node_id, expected_peers, num_connected_peers
```

Comment:
maybe also follow up diff, but we need to check full node network connectivity as well

Reply:
Actually, this is already being done since we use the same `launch_swarm` function in `LibraSwarm` to launch a full node swarm.